TODO.md

```
@@ -172,8 +172,16 @@
 - **✅ 統合ファクトリ**: TaskStorageFactory・SqliteCollaborationFacadeFactory
 - **✅ マイグレーション**: 6⇔3テーブル相互変換・データ保全機能
 
+### TaskAssignmentManager基盤 (100%完了・P3-1)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)